lib/generators/rolemodel/modals/modals_generator.rb

```diff
@@ -0,0 +1,43 @@
+module Rolemodel
+  class ModalsGenerator < Rails::Generators::Base
+    source_root File.expand_path('templates', __dir__)
+
+    def install_icons
+      generate 'rolemodel:css:base' unless File.exists?(Rails.root.join('app/javascript/packs/stylesheets.scss'))
+      generate 'rolemodel:css:icons'
+    end
+
+    def add_ui_components
+      copy_file 'app/helpers/dynamic_link_helper.rb'
+      addendum = File.read([source_paths.last, '/application_addendum.js'].join)
+      append_to_file 'app/javascript/packs/application.js', addendum
+      copy_file 'app/javascript/packs/rolemodel-custom-confirm.js'
+      copy_file 'app/javascript/packs/rolemodel-modal.js'
+      copy_file 'app/javascript/packs/rolemodel-panel.js'
```

Comment:
Doesn't `copy_file` need a destination path? Also do we really want to have all these files in the root of `packs` if so they will get emitted as JS bundles by webpack (I think). We don't ever intend to use them directly so it would be it would make sense for them to go in a sub folder

Reply:
If the file path the is the same as the destination, there is no needed to add a second argument.